backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -445,6 +445,69 @@ export class AnalyticsRepository {
       .execute();
   }
 
+  public async getCSVDataForWithInSubExport(
+    experimentId: string,
+    skip: number,
+    take: number
+  ): Promise<
+    Array<{
```

Comment:
Please declare a type instead of passing in a generic object definition. That way we can use that type where we are calling `any` in the other file.

Reply:
resolved